instagram/src/components/PostContainer/PostActions.js

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
-import { FiHeart, FiMessageCircle} from 'react-icons/fi';
+import { FiHeart, FiMessageCircle } from 'react-icons/fi';
 
 
 
 class PostActions extends React.Component {
     constructor(props){
         super(props);
         this.state = {
-            likes: this.props.likes
+            likes: this.props.likes,
+            likeToggle: false
         }
     }
 
-
     addLike = () => {
-        this.setState({ likes: this.state.likes + 1 })
+        if (this.state.likeToggle === false) {
```

Comment:
Really nice. Making it so that the user can only like a comment once, then if they click it again it unlikes. :) Great functionality.